docs/source/index.rst

```diff
@@ -15,16 +14,22 @@ original system of `JupyterHub`_. As of now, you can find two main cases:
 1. `The Littlest JupyterHub <https://github.com/jupyterhub/the-littlest-jupyterhub>`__ distribution is suitable if you need a small number of users (1-100) and a single server with a simple environment.
 2. `Zero to JupyterHub with Kubernetes <https://github.com/jupyterhub/zero-to-jupyterhub-k8s>`__ allows you to deploy dynamic servers on the cloud if you need even more users.
 
+JupyterHub supports web browsers, tablets and smartphones and can be used by 
```

Comment:
I'm not sure this is true - JupyterHub only really works well with full keyboard input. Where did you find this information?

Reply:
Hi! Whilst I was doing some research into the applicability of JupyterHub, I came across articles and YouTube videos detailing how to install and use JupyterHub on Android smartphones.
Would you suggest that I take out the "tablets and smartphones" entirely from the phrase?

Reply:
Yes, I would suggest removing the "JupyterHub supports..." bit entirely.

Reply:
This has been deleted.